web/src/components/trace/index.tsx

```diff
@@ -103,11 +104,13 @@ export function Trace(props: {
             trace={props.trace}
             observations={props.observations}
             scores={props.scores}
+            configs={props.configs}
```

Comment:
we do not need to pass the prop down, we could just fetch the config down in the annotation drawer as react query will cache it locally if is needed mutliple times within the same session